client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          We now support generic extension properties, which will be available for execution starting from Camunda Platform 8.1.
         </li>
         <li>
-          <h4>Diagram errors in the properties panel</h4>
-          All diagram errors are shown in the properties panel as of this release. Open the error panel for the errors to be shown.
+          <h4>Diagram errors on the canvas</h4>
+          Diagram errors are now shown on the canvas. Click on an error to show details.
+        </li>
+        <li>
+          <h4>Sticky properties panel headers</h4>
+          The section headers of the BPMN and DMN properties panels are now sticky to provide reliable context.</li>
+        <li>
+          <h4>Modeling UX improvements and bug fixes</h4>
+          This release includes a lot of modeling UX improvements. Sequence flows are now always displayed on top. Also, we fixed several issues to improve the copy and paste behavior.
```

Comment:
Do we actually have a lot of improvements?

Reply:
I'd say so, it includes almost everything from this list: https://github.com/bpmn-io/internal-docs/issues/607

Reply:
Mh, I don't know whether this one is worth a mention. Technically we simply upgraded the Codemirror version, and the search bar now has a bit more functionality. But I don't know whether we want to promote this one at all.

Looking at the current snapshot, we already include a lot of stuff (especially compared to the last release notes).

![image](https://user-images.githubusercontent.com/9433996/189133010-e6074ee6-67f2-460e-bdb2-a8d78889931a.png)


It's mentioned [in the Changelog](https://github.com/camunda/camunda-modeler/pull/3121).

Reply:
Agree